components/permit-holders/PermitHolderHeader.tsx

```diff
@@ -1,17 +1,18 @@
-import { Box, Flex, HStack, Text } from '@chakra-ui/react'; // Chakra UI
+import { Badge, Box, Flex, HStack, Text, Wrap } from '@chakra-ui/react'; // Chakra UI
 import { ChevronLeftIcon } from '@chakra-ui/icons'; // Chakra UI icon
 import Link from 'next/link'; // Link
-import RequestStatusBadge from '@components/internal/RequestStatusBadge'; // Request status badge
-import { Applicant, ApplicationStatus } from '@lib/graphql/types'; // Applicant type
+import { ApplicantStatus } from '@lib/graphql/types'; // Applicant type
+import React from 'react'; // React
+import { ApplicantData } from '@pages/admin/permit-holder/[permitHolderId]'; // Applicant data type
 
 type PermitHolderHeaderProps = {
-  readonly applicant: Applicant;
-  readonly applicationStatus: ApplicationStatus | 'EXPIRING' | 'EXPIRED' | 'ACTIVE';
+  readonly applicant?: ApplicantData;
+  readonly applicantStatus?: ApplicantStatus;
```

Comment:
We do not want to allow these props to be optional - if they were optional, then we could be rendering empty content here. To enforce loading states, we want to do conditional rendering at the page level, where this component is being used.